charts/newrelic-logging/k8s/new-relic-fluent-plugin.yml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: newrelic-logging
-  namespace: default
+  namespace: newrelic
   labels:
     k8s-app: newrelic-logging
     version: v1
```

Comment:
Please note this applies to both the K8s plugin and the Helm chart template.
- For the K8s plugin, I'd suggest to leave that line with just a `*` wildcard to avoid having to update it for each new version.
- For the Helm chart template, you can also use just the wildcard, or parametrize it with `{{ .Chart.AppVersion }}`. In this case, I think it makes sense to use the latter option.